ehrapy/api/tools/nlp/_translators.py

```diff
@@ -0,0 +1,252 @@
+                    print(
+                        "[bold yellow]Reached 90% of the character translation limit. "
+                        "Ensure that you have enough quota."
+                    )
+                    print(f"[bold yellow]{self.translator.get_usage}")
+                elif usage.document.limit is not None and (usage.document.count / usage.document.limit) > 0.9:
+                    print(
+                        "[bold yellow]Reached 90% of the document translation limit. "
+                        "Ensure that you have enough quota."
+                    )
+                    print(f"[bold yellow]Current usage: {usage.document.count}")
+                elif (
+                    usage.team_document.limit is not None
+                    and (usage.team_document.count / usage.team_document.limit) > 0.9
+                ):
+                    print(
+                        "[bold yellow]Reached 90% of the team document translation limit "
+                        "Ensure that you have enough quota"
+                    )
+                    print(f"[bold yellow]Current usage: {usage.team_document.count}")
+
+                return function(self, *args, **kwargs)  # type: ignore
+
+        return wrapper
+
+    @_check_usage  # type: ignore # pragma: no cover
+    def authenticate(self, authentication_key: str) -> None:
+        """Authenticates the DeepL user
+
+        Args:
+            authentication_key: DeepL authentication key
+        """
+        self.translator = deepl.Translator(authentication_key)
+
+    def print_source_languages(self) -> None:  # pragma: no cover
+        """prints all possible source languages to translate from
+
+        Example: "DE (German)"
+        """
+        for language in self.translator.get_source_languages():
+            print(f"{language.code} ({language.name})")
+
+    def print_target_languages(self) -> None:  # pragma: no cover
+        """Prints all possible target languages to translate to"""
+        for language in self.translator.get_target_languages():
+            if language.supports_formality:
+                print(f"{language.code} ({language.name}) supports formality")
+            else:
+                print(f"{language.code} ({language.name})")
+
+    @_check_usage  # type: ignore
+    def translate_text(self, text: Union[str, List], target_language: str) -> Union[TextResult, List[TextResult]]:
+        """Translates the provided text into the target language
+
+        Args:
+            text: The text to translate
+            target_language: The target language to translate the Text into, e.g. EN-GB
+
+        Returns:
+            A :class:`~deepl.TextResult` object
+        """
+        return self.translator.translate_text(text, target_lang=target_language)
+
+    @_check_usage  # type: ignore # pragma: no cover
+    def translate_document(
+        self, input_file_path: str, output_path: str, target_language: str, formality: str = Formality.DEFAULT
```

Comment:
General question concerning "target_language": Shouldn't this (at least default to) always be English? Since (at least for now I'd say) our standard language is English.